lively.morphic/text/morph.js

```diff
@@ -2492,10 +2492,6 @@ export class Text extends Morph {
       .insetBy(this.borderWidth);
     let charBounds = this.charBoundsFromTextPosition(pos);
 
-    // if no line wrapping is enabled we add a little horizontal offset so
-    // that characters at line end are better visible
-    if (!this.lineWrapping) charBounds = charBounds.insetByPt(pt(-20, 0));
-
```

Comment:
Ah, I see.
I never had problems with this line, but I am also not exactly sure what its purpose was? If you had a problem that is solved now, I would think its okay.

Reply:
Me neither, but removing it fixed a bunch of things! I will keep this change until I discover some unintended behavior.